Makefile

```diff
@@ -506,6 +506,17 @@ stop: ## docker-compose stop - stops (but preserves) the containers
 	@$(WITH_LOCAL_GRAPL_ENV)
 	docker-compose $(EVERY_COMPOSE_FILE) stop
 
+# This is a convenience target for our frontend engineers, to make the dev loop
+# slightly less arduous for grapl-web-ui/engagement-view development.
+# It will *rebuild* the JS/Rust grapl-web-ui dependences, and then 
+# restart a currently-running `make up` web ui allocation, which will then
+# retrieve the latest, newly-rebuilt Docker container.
+.PHONY: restart-web-ui
+restart-web-ui: build-engagement-view  ## Rebuild web-ui image, and restart web-ui task in Nomad
```

Comment:
Can you add a note saying that this will only work as expected as long as tag is equal to dev. If someone is incrementing a version ie 0.15.0 -> 0.16.0, it won't work